pkgs/by-name/pr/proton-mail/package.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, dpkg
+, electron
+}
+:
+stdenv.mkDerivation rec {
+  pname = "proton-mail";
```

Comment:
We should consider calling it the same as other distros: https://repology.org/project/protonmail-desktop/versions

Reply:
ack